src/node.cc

```diff
@@ -190,8 +190,8 @@ static node_module* modlist_builtin;
 static node_module* modlist_internal;
 static node_module* modlist_linked;
 static node_module* modlist_addon;
-static bool trace_enabled = false;
-static std::string trace_enabled_categories;  // NOLINT(runtime/string)
+
+std::string trace_enabled_categories;  // NOLINT(runtime/string)
```

Comment:
Is the lint exception really needed here?

Reply:
Apparently, yes. Not entirely sure why but it complains without it.